web/beacon-app/src/components/ui/SetupTenant/TenantHeader.tsx

```diff
@@ -10,7 +10,7 @@ export default function TenantHeader() {
       <p className="mt-10 text-3xl">
         You can start by selecting the <span className="font-bold">Quick Start</span> tenant on the{' '}
         <span className="font-bold">Starter Plan</span>. If you&#39;d like to customize your tenant
-        based on cloud provider(s) and regions, you&#39;ll have to select a <span>paid plan</span>.
+        based on cloud provider(s) and regions, you&#39;ll have to select a <span className="font-bold">paid plan</span>.
```

Comment:
The copy currently displays `paid plan` as a link, but since we don't currently have a link I made it bold for now.